lib/atat-iam-stack.ts

```diff
@@ -88,17 +90,6 @@ export class AtatIamStack extends cdk.Stack {
       statements: [new statement.Organizations().deny().allActions().onAllResources()],
     });
 
-    // Eventually, this should be restricted to the specific services in use for the application
-    // rather than allowing all IAM actions. This still theoretically will give anyone who can pass
-    // the role permission to do almost anything in the account; however, it at least requires that
-    // it happen via CloudFormation.
-    const cloudFormationExecutionRole = new iam.Role(this, "CloudFormationExecutionRole", {
```

Comment:
Assuming I'm tracking this correctly, it's really nice to let CodePipeline handle some of this complexity.

Reply:
In this case, it's actually CDKv2. This has been dead code for a few weeks now. The specifics for this role are now defined as part of the CDK Bootstrap stack. That becomes more useful especially when we move towards considering a cross-account setup.